cypress/integration/main_page_spec.js

```diff
@@ -60,26 +60,53 @@ describe('Dashboard Home Page', () => {
     });
 
     it('displays a compatible Cumulus API Version number', () => {
-      const apiVersionNumber = 'a.b.c';
+      const apiVersionNumber = 'a.b.c-d.e';
       cy.window().its('appStore').then((store) => {
         store.dispatch({
           type: API_VERSION,
           payload: { versionNumber: apiVersionNumber }
         });
 
-        cy.get('div[class=api__version]').should((apiVersionWrapper) => {
-          expect(apiVersionWrapper.first()).to.contain(apiVersionNumber);
-        });
+        // Using regex to match any version format like x.y.z-type.n
+        cy.get('.footer__version')
+          .find('.api__version')
+          .invoke('text')
+          .should('match', /API v\d+\.\d+\.\d+-[a-zA-Z]+\.\d+/);
+      });
+    });
+
+    it('displays metrics overview with default 24-hour time range', () => {
```

Comment:
I'm surprised to find new assertions like this as part of the React router update. Is this related?

Reply:
No, that's not in result of my ticket. There was a previous ticket regarding the time range change for the Metrics Overview. I ran into the same issue with the API version. A previous change to the component that wasn't updated in the test. So...the responsibility is mine now.

Reply:
Ah, bummer.